docs/installation/upgrade/2.3-to-2.4.md

```diff
@@ -21,6 +21,44 @@ setup.
 
 ## Dovecot CE
 
+### Configuration Changes
+
+::: warning
+Dovecot 2.3.x settings will NOT work unless the configuration is changed
+as described in this section.
+:::
+
+<!-- @include: include/2.4-directory-hashing.inc -->
+
+<!-- @include: include/2.4-unknown-invalid-variables.inc -->
+
+<!-- @include: include/2.4-section-naming.inc -->
+
+<!-- @include: include/2.4-empty-userdb-variables.inc -->
```

Comment:
This doesn't seem a very important change. It would be relevant only if someone relies on the empty value -> yes translation. And now with the redesigned passdb/userdb it's even more difficult to do this. I'm thinking lets remove this.

Reply:
it's in the original documentation that has already been released though.